### ISSUE TITLE
Unable to locate package libgdbm5

### ISSUE THREAD

**Issue by europ** [rbenv/ruby-build#1211]
Section [`Suggested build environment`](https://github.com/rbenv/ruby-build/wiki#suggested-build-environment) at [`Wiki`](https://github.com/rbenv/ruby-build/wiki) contains an invalid package for Ubuntu 16.04.4.

command:
```
sudo apt-get install autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm5 libgdbm-dev
```

result:
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libgdbm5
```

**Comment by A-Boudi** [comment]
libgdbm5 is for Ubuntu 18.04. If you're using 16.04 you should install libgdbm3.

Link:
https://packages.ubuntu.com/search?keywords=libgd

**Comment by europ** [comment]
@A-Boudi Thank you, but I have it installed already. I opened this issue to to warn of an error in the [`Suggested build environment`](https://github.com/rbenv/ruby-build/wiki#suggested-build-environment).

**Comment by waiting-for-dev** [comment]
I  [added a notice](https://github.com/rbenv/ruby-build/wiki/Home/3e1797d58357f2618101e9632f048633f2150f5b) in the wiki about that. Maybe with that we could close this issue.

**Comment by AlexWayfer** [comment]
`libgdbm6` is available for Debian (10), and it's acceptable for `ruby-build`.

**Comment by shivabhusal** [comment]
`libgdbm5` is available for mint19 and ubuntu 19

**Comment by singhvaibhavram** [comment]
libgdbm6 is available for Ubuntu: 20.04

**Comment by singhvaibhavram** [comment]
`libgdbm6` is available for Ubuntu: 20.04

**Comment by MeetChavda** [comment]
It's version issue....
`sudo apt-get install autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm5 libgdbm-dev`

use `libgdbm6` instead of `libgdbm5`
